ci/matrix.xml

```diff
@@ -238,7 +238,7 @@
     sbt ++2.10.6 ir/publishLocal tools/publishLocal jsEnvs/publishLocal \
```

Comment:
I guess you can also remove `jasmineTestFramework/publishLocal` 2 lines above this one, then.